commands/password-managers/bitwarden/copy-first-matching-password.sh

```diff
@@ -46,15 +50,11 @@ if [ $unlocked_status -ne 0 ]; then
 fi
 
 item=$(bw list items --search $1 $session 2> /dev/null | jq ".[0] | { name: .name, password: .login.password }")
-name=$(echo $item | jq ".name")
-password=$(echo $item | jq --raw-output ".password")
-
-if [[ -z $name || -z $password ]]; then
-  echo "The query '$1' did not return a password."
-  exit 1
-fi
+name=$(echo $item | jq -e ".name") || notFoundError
+password=$(echo $item | jq -re ".password") || notFoundError
```

Comment:
When available, it's always a best practice to use the long-form of command options when coding them into a script. This preserves clarity, and in this case there aren't enough options to require the commands to overflow onto additional lines.
```suggestion
name=$(echo $item | jq --exit-status ".name") || notFoundError
password=$(echo $item | jq --raw-output --exit-status ".password") || notFoundError
```